### DISCUSSION TITLE
Need a logo for rspack.config.js

### DISCUSSION THREAD

**Discussion by justforuse** [web-infra-dev/rspack#3773] [General]
[vscode-icons](https://github.com/vscode-icons/vscode-icons/wiki/ListOfFiles) support set specific icon for file. but I do not find the proper icon for rspack, who can provide this? 😎

like `vite` or `vitest`, they both have icon for their config file:

<img width="975" alt="image" src="https://github.com/web-infra-dev/rspack/assets/11868477/cfd1a442-ff20-4601-a79b-456ae121a4b5">

**Comment by ScriptedAlchemy** [comment]
What format do you need the icon in? svg/transparent png?

**Reply by justforuse** [reply]
I think is `svg` file, like this:

https://github.com/vscode-icons/vscode-icons/blob/master/icons/file_type_vite.svg
